mappings/net/minecraft/util/math/random/Random.mapping

```diff
@@ -54,6 +54,8 @@ CLASS net/minecraft/class_5819 net/minecraft/util/math/random/Random
 	METHOD method_43057 nextFloat ()F
 	METHOD method_43058 nextDouble ()D
 	METHOD method_43059 nextGaussian ()D
-	METHOD method_43385 nextPredictable (DD)D
-		ARG 1 base
-		ARG 3 variance
+	METHOD method_43385 nextTriangular (DD)D
+		COMMENT {@return a random {@code double} between {@code mode - deviation} and
+		COMMENT {@code mode + deviation} (both inclusive) with mode {@code mode}}
+		ARG 1 mode
```

Comment:
The `mode` parameter should be named `mean` to align with other Yarn parameter names. For example, the `MathHelper.nextGaussian` method has `mean` and `deviation` parameters.

```suggestion
		COMMENT {@return a random {@code double} between {@code mean - deviation} and
		COMMENT {@code mean + deviation} (both inclusive) with mean {@code mean}}
		ARG 1 mean
```

Reply:
It's simultaneously mode and mean.

Reply:
I think that Yarn should standardize on using mean.

Reply:
Or just `center` as this is the center of the triangular distribution, more straightforward to general users

Reply:
Triangular distribution in general does not have to be symmetric, and in that case the peak is not the mean. Here both works but this is fine.